hw/ip/csrng/doc/checklist.md

```diff
@@ -49,15 +49,15 @@ RTL           | [FEATURE_COMPLETE][]    | Done        |
 RTL           | [AREA_CHECK][]          | Done        |
 RTL           | [PORT_FROZEN][]         | Done        |
 RTL           | [ARCHITECTURE_FROZEN][] | Done        |
-RTL           | [REVIEW_TODO][]         | Pending     |
+RTL           | [REVIEW_TODO][]         | Done        |
 RTL           | [STYLE_X][]             | Done        |
 Code Quality  | [LINT_PASS][]           | Done        |
```

Comment:
Even though not strictly required, it would be nice to get the verilator lint warnings diagnosed/sorted out as well.

https://reports.opentitan.org/hw/ip/csrng/lint/verilator/latest/results.html

I'd especially be interested in a diagnosis for 


%Warning-UNOPTFLAT: ../src/lowrisc_ip_aes_0.6/rtl/aes_cipher_control.sv:563:42: Signal unoptimizable: Feedback to clock or circular logic: 'csrng.u_csrng_core.u_csrng_block_encrypt.u_aes_cipher_core.u_aes_cipher_control.sp2v_sig'


Which in the past was a good indication of something going wrong (but sometimes it's a false alarm as well).

Reply:
@vogelpi what do you think of the verilator by @imphil ?

Reply:
The UNOPTFLAT warning can be waived. Verilator erroneously sees loops involving these sparsely encoded signals. Inside the AES IP, the warning occurs in the main controller which is not included inside CSRNG. Thus, I would recommend to waive it locally inside CSRNG.

Reply:
That sound fine to me. Verilator is not easy for us to run, just as ascentlint is not accessable for other. Anyway you can supply the waiver statement? I will then commit it.

Reply:
Oh, I see. Let's do the following: I'll help you out with the ones inside AES and the primitives (will do a PR for those) but I would need you to have a look at all warnings and errors and see if any of these are critical.

We can do the same also for ES and EDN if you like. Let's follow up offline.

Reply:
AES looks clean for ascentlint currently. Most verilator warnings/errors are in the common modules, but not sure whether they should be fixed or waived, with the former being the challenge. Yes, let's circle back on this and come up with a plan.

Reply:
@tjaychen since @vogelpi is addressing this last concern in #5965, I think we can merge this issue now.